core_v2/sources/domains.move

```diff
@@ -489,13 +488,15 @@ module aptos_names_v2::domains {
             );
         };
 
-        let reverse_lookup_result = get_reverse_lookup(account_addr);
-        if (option::is_none(&reverse_lookup_result)) {
-            // If the user has no reverse lookup set, set the user's reverse lookup.
-            set_reverse_lookup(sign, subdomain_name, domain_name);
-        } else if (!is_subdomain(subdomain_name)) {
-            // Automatically set the name to point to the sender's address
-            set_target_address_internal(subdomain_name, domain_name, signer::address_of(sign));
```

Comment:
`set_target_address_internal` could be replace by [`set_target_address`](https://github.com/aptos-labs/aptos-names-contracts/blob/bl/rew/core_v2/sources/domains.move#L873)  because `set_target_address` does 2 more things beside `set_target_address_internal`, 1. it will clear the reverse lookup if it's a domain (but it's not applicable here, cause the new name doesn't have a target address yet,  2. clear signer's reverse lookup if signer's reverse lookup is the name registering now, this is also not applicable cause we always set target address first before set reverse lookup

Reply:
however this is pretty dangerous, let me know if you guys think i should make this change `set_target_address_internal` -> `set_target_address`